wagtail/wagtail
# Convert icon SVG root elements from <symbol> to <svg>

This executes the first two steps of #7511, changing our icon SVG files to have a root element of `<svg>` instead of `<symbol>`, which will make them easier to use in a CSS context, and make them generally easier to work with.

The Python that puts all of the icons together into the inline sprite is updated to replace `svg` with `symbol` on the fly so that our existing method of referencing icons within the sprite works the same as before.

_Please check the following:_

    - ~~[ ] **Please list the exact browser and operating system versions you tested**:~~
    - ~~[ ] **Please list which assistive technologies you tested**: ~~

The resulting rendered markup for the icon sprite should be the same as before.


## How to test

Run this branch in any testing environment and confirm that SVG icons load as they did previously (menu and dashboard stats are the easiest ones to see).
